packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -237,6 +240,13 @@ export const _isTeamsCall: (call: CallCommon) => call is TeamsCall_2;
 // @internal (undocumented)
 export const _isTeamsCallAgent: (callAgent: CallAgentCommon) => callAgent is TeamsCallAgent_2;
 
+// @beta
+export interface LocalRecordingCallFeatureState {
+    activeRecordings?: LocalRecordingInfo[];
```

Comment:
can there be multiple local recordings active at same time? If so, `isRecordingActive` points to which recording?

Reply:
Yes, not available yet in Teams but it is in the roadmap so we will need to be scalable.
isRecordingActive points to ANY instance of local recording that is active. If any participant is actively recording then the state will reflect true.